ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -657,22 +679,111 @@ public void shouldInsertInto() throws Exception {
   }
 
   @Test
-  public void shouldHandleErrorResponseFromInsertInto() {
+  public void shouldPullQueryTableAfterInsert() throws Exception {
```

Comment:
A regular table should _not_ be pull-queryable? Is the test name incorrect?

Overall, I am wondering what you try to do: insert into the table and verify that the data was written? For this case you could use a push query in the test.

Reply:
Yeah, test name should be `shouldNot...` 

I'm trying to test you can insert into table created using `with` but pull query it will throw exception

Reply:
My question is really: this PR is about allowing INSERT INTO for tables -- however, INSERT INTO and pull-queries have nothing to do with each other. So what do we gain by issuing a pull query (that we know is not supported for this case anyway)? -- Overall, it seems that the second part of this test is unnecessary and the first part is covered by the next test that uses a push query to verify if the insert was successful. Thus, should we remove this test?